packages/core/src/components/button/_button-group.scss

```diff
@@ -147,7 +146,6 @@ Styleguide components.button-group.css
 
       &::after {
         @include pt-button-minimal-divider();
-
```

Comment:
I get what's forbidden, simple new lines before rule declaration. Maybe an other option would be to move those `@include`s at the bottom of all the rules? With a new line even maybe? Or do as you suggested, just remove that new line everywhere but I'm gonna miss it 😢